packages/nuxt/src/composables.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  useGlobalConfig as useGlobalConfigLocal,
+  getGlobalConfig as getGlobalConfigLocal,
+  setGlobalConfig as setGlobalConfigLocal,
+  mergeGlobalConfig as mergeGlobalConfigLocal,
+  useColors as useColorsLocal,
+  getColor as getColorLocal,
+  getColors as getColorsLocal,
+  setColors as setColorsLocal,
+  GlobalConfig,
+  GlobalConfigUpdater,
+  ColorConfig
+} from 'vuestic-ui'
```

Comment:
```suggestion
} from 'vuestic-ui/src/main'
```

Reply:
Nuxt module is independent package, that we likely shouldn't even bundle. So this whole module using vuestic-ui from user's node_modules, not our repo.